fedavg.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import tensorflow as tf
+from tensorflow_federated.python.learning import federated_averaging
+import os
+import asyncio
+from fastapi import FastAPI, WebSocket, File, UploadFile
+from fastapi.responses import HTMLResponse
+import websockets
+import copy
+
+app = FastAPI()
+
+datasets = []
+num_clients = 1
+model = None
+clients = [0] #this would be a list of all client ids
+
+@app.get('/')
+async def root:
+    return 'root'
+
+@app.get("/{model}/")
+async def update_model(model: tf.keras.Model):
+    return model.load_weights('consolidated.h5')
```

Comment:
return a url that can be retrieved from the server.
```suggestion
    return FileResponse('consolidated.h5')
```